src/pages/Api/swagger.json

```diff
@@ -0,0 +1,8 @@
+{
+    "openapi": "3.0.1",
+    "info": {
+        "title": "The content will be replaced when we deploy.",
```

Comment:
This file will be replaced by auto-generate content.

Reply:
Replaced by `This file will be replaced by auto-generated content.`